src/controller/CHIPDeviceController.cpp

```diff
@@ -1631,6 +1649,57 @@ void AttributeReadFailure(void * context, CHIP_ERROR status)
     commissioner->CommissioningStageComplete(status);
 }
 
+void DeviceCommissioner::OnDone()
+{
+    NetworkClusters clusters;
```

Comment:
Should null out mReadClient here so it can get destroyed, right?

Reply:
done.